docker/oss-fuzz-builder/Dockerfile

```diff
@@ -43,4 +45,11 @@ RUN touch $SRC/__init__.py
 ENV ASAN_OPTIONS="detect_leaks=0"
 
 ENV OSS_FUZZ 1
+
+ENV CHECKOUT_COMMIT $checkout_commit
+ENV CHECKOUT_COMMIT_REPO_PATH $checkout_commit_repo_path
+RUN mkdir /opt/fuzzbench/
```

Comment:
Why need a dir at all, can copy to / ?

Reply:
Seems much cleaner to put things in a specific location rather than `/`?